andela/suyabay
# Carry out some user interface fixes 120945353

#### What does this PR do?

This PR handles some of the UI fixes on the application.
#### Description of Task to be completed?

The logout link is placed in the top dropdown menu.
The 'View more comments' now shows only when there are more comments to show.
Guest users can also view more comments.
#### How should this be manually tested?

> As a guest or authenticated user, visit an episode's route, e.g [http://suyabay-staging.herokuapp.com/episodes/32](http://suyabay-staging.herokuapp.com/episodes/32).
> You should see the `View more comments` button, only if there are more comments under that episode, 
#### Any background context you want to provide?

Initially guest users could not view other comments. This has been fixed.

The logout link is tied to the current authenticated user and is therefore placed as part of the dropdown menu list.
#### What are the relevant pivotal tracker stories?

The related PT story can be viewed [here](https://www.pivotaltracker.com/story/show/120945353).
#### Screenshots (if appropriate)
#### Questions:

There are no questions at this moment.

[@unicodeveloper](https://github.com/unicodeveloper)
